src/lib/storage/fixed_string_dictionary_segment/fixed_string_vector.hpp

```diff
@@ -58,14 +58,16 @@ class FixedStringVector {
   ReverseIterator rend() noexcept;
 
   // Return a pointer to the underlying memory
-  char* data();
+  const char* data() const;
```

Comment:
As I just had to look this up again: Do you know cdecl.org?

Reply:
Didn't know this :D
The reason for adding this was that I guessed that we do not want to be able to change the data when accessing `data()` and then `const char *` is required.

Reply:
Yepp, completely correct.